generators/service-discovery/index.js

```diff
@@ -1,13 +1,11 @@
-'use strict'
+'use strict';
 const BaseGenerator = require('../lib/generatorbase');
-
 const SCAFFOLDER_PROJECT_PROPERTY_NAME = "discovery";
-const SERVICE_NAME = "service-watson-discovery";
```

Comment:
The Watson Java Spring sdk will be expecting the search patterns in the `mappings.json` file to be `watson.discovery.url` or `watson_discovery_url` not `discovery_url`, see https://github.com/watson-developer-cloud/java-sdk/wiki/Watson-Spring-Boot-Support